dashboard/app/models/census/census_summary.rb

```diff
@@ -91,4 +97,290 @@ def self.merge_from_csv(filename, options = CSV_IMPORT_OPTIONS, is_dry_run: fals
+      submissions_summary[:counts][:teacher_or_admin],
+      result,
+      details
+    )
+
+    result = conditional_result(
+      'inconsistent_teacher_surveys',
+      submissions_summary[:has_inconsistent_surveys][:teacher_or_admin],
+      'MAYBE',
+      submissions_summary[:counts][:teacher_or_admin],
+      result,
+      details
+    )
+
+    result = conditional_result(
+      'non_teacher_surveys',
+      submissions_summary[:consistency][:not_teacher_or_admin] || submissions_summary[:has_inconsistent_surveys][:not_teacher_or_admin],
+      submissions_summary[:consistency][:not_teacher_or_admin] || 'MAYBE',
+      submissions_summary[:counts][:not_teacher_or_admin],
+      result,
+      details
+    )
+
+    result = conditional_result(
+      'last_years_summary',
+      map_historical_teaches_cs(previous_years_results.first),
+      map_historical_teaches_cs(previous_years_results.first),
+      previous_years_results.first,
+      result,
+      details,
+      true
+    )
+
+    result = conditional_result(
+      'two_years_agos_summary',
+      map_historical_teaches_cs(previous_years_results.second),
+      map_historical_teaches_cs(previous_years_results.second),
+      previous_years_results.second,
+      result,
+      details,
+      true
+    )
+
+    audit[:explanation] = details
+    return result
+  end
+
+  def self.summarize_school_data(summarization_data)
+    school = summarization_data[:school]
+    school_years = summarization_data[:school_years]
+
+    summaries = []
+    previous_years_results = []
+
+    school_years.each do |school_year|
+      audit = empty_audit_data
+      stats = get_school_stats(school, audit)
+
+      submissions_summary = summarize_submission_data(
+        school,
+        school_year,
+        stats[:high_school],
+        stats[:k8_school],
+        audit
+      )
+
+      summary = Census::CensusSummary.find_or_initialize_by(
+        school: school,
+        school_year: school_year,
+      )
+      summary.teaches_cs = compute_teaches_cs(
+        submissions_summary,
+        previous_years_results,
+        audit
+      )
+
+      audit[:previous_years_results] = previous_years_results
+      previous_years_results.unshift summary.teaches_cs
+
+      summary.audit_data = JSON.generate(audit)
+      summaries.push summary
+    end
+
+    return summaries
+  end
+
+  def self.summarize_census_data
+    school_years = (2016..SharedConstants::CENSUS_CONSTANTS.CURRENT_CENSUS_SCHOOL_YEAR)
+
+    ActiveRecord::Base.transaction do
+      School.eager_load(school_info: :census_submissions).
+        eager_load(:school_stats_by_year).
+        find_each do |school|
+          summarize_school_data(
+            {
+              school: school,
+              school_years: school_years,
+            }
+          ).each do |summary|
+            if block_given?
+              yield summary
+            else
+              summary.save!
```

Comment:
apologies, it looks like I may have made an important omission in the work plan. the idea is that this cronjob should be uploading what's already in the table, exactly as provided by the seed_all task you recently added. what I'm seeing here is writing of new data to the census_summaries table, which looks like the old algorithm which we have gotten rid of and do not want to bring back. 

I may be missing something, but it looks like most or all of the code added to this file can be removed.